smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -78,6 +78,15 @@ protected async Task GetOrCreateEdgeDeviceIdentity()
             {
                 await this.CreateEdgeDeviceIdentity(rm);
             }
+
+            //Adding a GetDeviceAsync again so it flushes the cash.
+            Device checkingDevice = await rm.GetDeviceAsync(this.deviceId);
```

Comment:
Did we confirm that this is needed and that it helps us?

Reply:
I don't think there is a way to confirm that. After I added the tests started passing. I can remove in the future. I also created a task under the e2e pbi so I can add to quickstart the capability to turn debug log on. That will help us in the future to debug.

Reply:
Actually I will step back and remove that code. 

If tests keep failing I will investigate. Better than having a code that we don't need.